Tests/Packet++Test/Tests/FtpTests.cpp

```diff
@@ -90,174 +90,179 @@ PTF_TEST_CASE(FtpParsingTests)
-		{pcpp::FtpRequestLayer::FtpCommand::PBSZ, "Protection Buffer Size"},
-		{pcpp::FtpRequestLayer::FtpCommand::PORT, "Specifies an address and port to which the server should connect"},
-		{pcpp::FtpRequestLayer::FtpCommand::PROT, "Data Channel Protection Level"},
-		{pcpp::FtpRequestLayer::FtpCommand::PWD, "Print working directory. Returns the current directory of the host"},
-		{pcpp::FtpRequestLayer::FtpCommand::QUIT, "Disconnect"},
-		{pcpp::FtpRequestLayer::FtpCommand::REIN, "Re initializes the connection"},
-		{pcpp::FtpRequestLayer::FtpCommand::REST, "Restart transfer from the specified point"},
-		{pcpp::FtpRequestLayer::FtpCommand::RETR, "Retrieve a copy of the file"},
-		{pcpp::FtpRequestLayer::FtpCommand::RMD, "Remove a directory"},
-		{pcpp::FtpRequestLayer::FtpCommand::RMDA, "Remove a directory tree"},
-		{pcpp::FtpRequestLayer::FtpCommand::RNFR, "Rename from"},
-		{pcpp::FtpRequestLayer::FtpCommand::RNTO, "Rename to"},
-		{pcpp::FtpRequestLayer::FtpCommand::SITE,
-		 "Sends site specific commands to remote server (like SITE IDLE 60 or SITE UMASK 002). Inspect SITE HELP "
-		 "output for complete list of supported commands"},
-		{pcpp::FtpRequestLayer::FtpCommand::SIZE, "Return the size of a file"},
-		{pcpp::FtpRequestLayer::FtpCommand::SMNT, "Mount file structure"},
-		{pcpp::FtpRequestLayer::FtpCommand::SPSV, "Use single port passive mode (only one TCP port number for both "
-												  "control connections and passive-mode data connections)"},
-		{pcpp::FtpRequestLayer::FtpCommand::STAT,
-		 "Returns information on the server status, including the status of the current connection"},
-		{pcpp::FtpRequestLayer::FtpCommand::STOR, "Accept the data and to store the data as a file at the server site"},
-		{pcpp::FtpRequestLayer::FtpCommand::STOU, "Store file uniquely"},
-		{pcpp::FtpRequestLayer::FtpCommand::STRU, "Set file transfer structure"},
-		{pcpp::FtpRequestLayer::FtpCommand::SYST, "Return system type"},
-		{pcpp::FtpRequestLayer::FtpCommand::THMB, "Get a thumbnail of a remote image file"},
-		{pcpp::FtpRequestLayer::FtpCommand::TYPE, "Sets the transfer mode (ASCII/Binary)"},
-		{pcpp::FtpRequestLayer::FtpCommand::USER, "Authentication username"},
-		{pcpp::FtpRequestLayer::FtpCommand::XCUP, "Change to the parent of the current working directory"},
-		{pcpp::FtpRequestLayer::FtpCommand::XMKD, "Make a directory"},
-		{pcpp::FtpRequestLayer::FtpCommand::XPWD, "Print the current working directory"},
-		{pcpp::FtpRequestLayer::FtpCommand::XRCP, ""},
-		{pcpp::FtpRequestLayer::FtpCommand::XRMD, "Remove the directory"},
-		{pcpp::FtpRequestLayer::FtpCommand::XRSQ, ""},
-		{pcpp::FtpRequestLayer::FtpCommand::XSEM, "Send, mail if cannot"},
-		{pcpp::FtpRequestLayer::FtpCommand::XSEN, "Send to terminal"}};
-
-	for (const auto &entry : possibleCommandCodes)
+		{ static_cast<pcpp::FtpRequestLayer::FtpCommand>(0), "Unknown command"		                                                                          },
```

Comment:
It doesn't format it nicely - add redundant spaces at the end of each line, and sometimes add redundant tabs. Let's format it properly and ask clang-format to ignore?

Reply:
fixed.